### DISCUSSION TITLE
FloatLabel with unstyled and tailwind - input and placeholder positioning bug

### DISCUSSION THREAD

**Discussion by baracuda69** [primefaces/community#1310] [PrimeVue] [answered]
The input position, (and the placeholder) are bugged - the placeholder is in the same position as the icon, so they overlap.
Additionally, if you enter text to the input and then lose focus on the input, the floating label does not stay above the input (floating) - it returns back to it's original position and merges with both the icon and the input-value.

PrimeVue configured in Nuxt3

//nuxt.config.ts
primevue: {
    options: { unstyled: true },
    importPT: { from: '~/assets/css/primevue-presets/lara/' },
  }

I have prepared a sample on stackblitz.com so you can easily see the issue;
[https://stackblitz.com/edit/nuxt-starter-ocxyua?file=app.vue](stackblitz example)

Note that I have pre-filled in the InputText and Password fields with values, so the issue is immediately visible.

Finally, is there a way to use the Password component inside IconField?
As you can see, the icon is not inside the Password like it is with InputText component.

**Comment by tugcekucukoglu** [comment, marked as answer]
Could you create an issue for this? https://github.com/primefaces/primevue-tailwind

Thank you!

**Reply by fotisskal** [reply]
Hello @tugcekucukoglu and @baracuda69 , is there already a ticket for this? Because I could not find anything relevant.
